Sources/SwiftDocC/Infrastructure/External Data/OutOfProcessReferenceResolver.swift

```diff
@@ -51,8 +51,13 @@ import SymbolKit
 public class OutOfProcessReferenceResolver: ExternalDocumentationSource, GlobalExternalSymbolResolver {
     private let externalLinkResolvingClient: ExternalLinkResolving
     
+    @available(*, deprecated, renamed: "id", message: "Use 'id' instead. This deprecated API will be removed after 6.2 is released")
+    public var bundleIdentifier: String {
+        id.rawValue
+    }
+    
     /// The bundle identifier for the reference resolver in the other process.
-    public let bundleIdentifier: String
+    public let id: DocumentationBundle.Identifier
```

Comment:
I can't name it `bundleIdentifier` because there's already a property with that name but I can name it `bundleID`.

Like I said in [this comment](https://github.com/swiftlang/swift-docc/pull/1069#discussion_r1851774625) I'm planning on deprecating the term "bundle" because it's causing confusion and I was trying to avoid needing to introduce and deprecate a public API in the same release but I don't think there's would be too many places like this, so perhaps that's not too bad.